src/components/AllPools/styles.scss

```diff
@@ -0,0 +1,9 @@
+.AllPools {
+  &__row {
+    display: flex;
+  }
+
+  &__adjustedRow {
+    width: 15%;
+  }
```

Comment:
not sure if there is a nicer way to do this. There was a huge gap between the 3 and 4 column

Reply:
I think in this specific case setting width is OK. It definitely looks better than with the gap. 👍